test/snapper_dbus_escape_test.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env rspec
+# coding: utf-8
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+Yast.import "Snapper"
+
+
+describe "SnapperDbus#escape" do
+
+
+  it "call with nothing special" do
```

Comment:
This is not much RSpec-like. You must describe the result of calling the method. Anyway, these tests are simple enough so feel free to leave them as they are.